mule-artifact-it/src/test/java/integrationTests/mojo/environmentSetup/StandaloneEnvironment.java

```diff
@@ -236,4 +237,14 @@ public void setMuleHome(String muleHome) {
   public String getMuleExecutable() {
     return muleHome + EXECUTABLE_FOLDER_RELATIVE_PATH;
   }
+
+  public void runStandalone() throws IOException, InterruptedException {
+    unpackAgent();
+    startMule();
+    checkStandaloneIsAcceptingDeployments();
+  }
+
+  private void checkStandaloneIsAcceptingDeployments() throws InterruptedException, IOException {
```

Comment:
This checking is used in the before method of the standalone IT. I changed the method implementation to throw a TimeoutException given a number of seconds. I also added a (timeout = 60000) to the test annotation